tests/cloudpickle_test.py

```diff
@@ -440,6 +442,58 @@ def method(self, x):
         mod1, mod2 = pickle_depickle([mod, mod])
         self.assertEqual(id(mod1), id(mod2))
 
+    def test_dynamic_modules_globals(self):
+        # _dynamic_modules_globals is a WeakValueDictionary, so if this dynamic
+        # module has no other reference than in this # dict (whether on the
+        # child process or the parent process), this module will be garbage
+        # collected.
```

Comment:
In the context of this test, `_dynamic_modules_globals` is only used in the child process.

Also it's not the module that gets garbage collected by the function globals dict.